plugins/CMakeLists.txt

```
@@ -62,6 +62,9 @@ MACRO(buildplugin TARGET TARGETLIB)
     set_target_properties(${TARGET} PROPERTIES LINK_FLAGS "/INCREMENTAL:NO /DEBUG /OPT:REF /OPT:ICF")
     # Set file name & location
     set_target_properties(${TARGET} PROPERTIES COMPILE_PDB_NAME ${TARGET} COMPILE_PDB_OUTPUT_DIR ${CMAKE_BINARY_DIR})
+    IF(MPI_C_INCLUDE_PATH)
+      target_include_directories(${TARGET} PRIVATE ${MPI_C_INCLUDE_PATH})
+    ENDIF(MPI_C_INCLUDE_PATH)
```

Better to use `target_link_libraries` with the MPI target to make sure this gets linked against the libraries if necessary:

```suggestion
    IF(TARGET MPI::MPI_C)
      target_link_libraries(${TARGET} PRIVATE MPI::MPI_C)
    ENDIF(TARGET MPI::MPI_C)
```